piff/config.py

```diff
@@ -191,3 +191,122 @@ def plotify(config, logger=None):
     for stats in output.stats_list:
         stats.compute(psf,stars,logger=logger)
         stats.write(logger=logger)
+
+def meanify(config, logger=None):
+    """Take Piff output(s), build an average of the FoV, and wirte output average.
+
+    :param config:      The configuration file that defines how to build the model
+    :param logger:      A logger object for logging progress. [default: None]
+    """
+    from .input import Input
+    from .psf import PSF
+    from .star import Star
+    from .output import Output
+    import glob
+    import numpy as np
+    import fitsio
+
+    if logger is None:
+        verbose = config.get('verbose', 1)
+        logger = setup_logger(verbose=verbose)
+
+    for key in ['input', 'output']:
+        if key not in config:
+            raise ValueError("%s field is required in config dict"%key)
+    for key in ['file_name']:
+        if key not in config['input']:
+            raise ValueError("%s field is required in config dict output"%key)
+
+    for key in ['file_name']:
+        if key not in config['output']:
+            raise ValueError("%s field is required in config dict output"%key)
+
+    if 'dir' in config['input']:
+        dir = config['input']['dir']
+    else:
+        dir = None
+
+    if 'binning' in config['output']:
+        if type(config['output']['binning']) is not int:
+            ValueError("type of binning should be integer (%s now)"%type(config['output']['binning']))
```

Comment:
"type of binning" being an integer seems incongruous.  Probably this should be `nbins`?  

Or, probably better, this should be given as bin_spacing in arcsec, rather than nbins.  People may run this over single ccds or full fields or even partial fields probably.  Also, not all possible fields are the same size in x and y (e.g. if running on single chips -- they could be 2K x 4K.)  So specifying a physical size of the cells in which to average things is probably more useful than specifying a number of bins across the field.

Reply:
I totally agree for that. However, the coordinate u & v, are they all the time in arcsec ? If yes, there is no problem for me to implement the binning in physical units.

Reply:
Yes, u,v are defined to be in arcsec.  (Unless someone inputs a pathological WCS I suppose, but let's not worry about that.)